test/mocks/geek_jokes_responses.js

```diff
@@ -0,0 +1,22 @@
+const nock = require('nock');
+
+const {
+  common: {
+    geekJokesApi: { baseUrl }
+  }
+} = require('../../config');
+
+const exampleJoke = 'When Chuck Norris makes a burrito, its main ingredient is real toes.';
+const { origin, pathname } = new URL(baseUrl);
+
+exports.mockGeekJokesSuccessResponse = (joke = exampleJoke) =>
+  nock(origin)
+    .get(pathname)
+    .query({ format: 'json' })
+    .reply(200, { joke });
+
+exports.mockGeekJokesFailureResponse = () =>
+  nock(origin)
+    .get(pathname)
+    .query({ format: 'json' })
+    .reply(500);
```

Comment:
This two functions could be converted into a 
`mockGeekJokeWithResponse = response => {...}`